scripts/exregional_process_bufr.sh

```diff
@@ -0,0 +1,382 @@
+BUFR_TABLE=${FIXgsi}/prepobs_prep_RAP.bufrtable
+cp_vrfy $BUFR_TABLE prepobs_prep.bufrtable
+
+#-----------------------------------------------------------------------
+#
+#   set observation soruce 
+#
+#-----------------------------------------------------------------------
+SUBH=""
+obs_source=rap
+if [[ ${HH} -eq '00' || ${HH} -eq '12' ]]; then
+  obs_source=rap_e
+fi
+
+case $MACHINE in
+
+"WCOSS2")
+
+  obsfileprefix=${obs_source}
+  obspath_tmp=${OBSPATH}/${obs_source}.${YYYYMMDD}
+
+  ;;
+*)
+
+  obsfileprefix=${YYYYMMDDHH}.${obs_source}
+  obspath_tmp=${OBSPATH}
+
+esac
+
+#
+#-----------------------------------------------------------------------
+#-----------------------------------------------------------------------
+#
+# Link to the observation lightning bufr files
+#
+#-----------------------------------------------------------------------
+
+obs_file=${obspath_tmp}/${obsfileprefix}.t${HH}${SUBH}z.lghtng.tm00.bufr_d
+print_info_msg "$VERBOSE" "obsfile is $obs_file"
+run_lightning=false
+if [ -r "${obs_file}" ]; then
+   cp_vrfy "${obs_file}" "lghtngbufr"
+   run_lightning=true
+else
+   print_info_msg "$VERBOSE" "Warning: ${obs_file} does not exist!"
+fi
+
+
+#-----------------------------------------------------------------------
+#
+# Build namelist and run executable for lightning
+#
+#   analysis_time : process obs used for this analysis date (YYYYMMDDHH)
+#   minute        : process obs used for this analysis minute (integer)
+#   trange_start  : obs time window start (minutes before analysis time)
+#   trange_end    : obs time window end (minutes after analysis time)
+#   bkversion     : grid type (background will be used in the analysis)
+#                   0 for ARW  (default)
+#                   1 for FV3LAM
+#-----------------------------------------------------------------------
+
+if [[ "$run_lightning" == true ]]; then
+
+  cat << EOF > namelist.lightning
+ &setup
+  analysis_time = ${YYYYMMDDHH},
+  minute=00,
+  trange_start=-10,
+  trange_end=10,
+  grid_type = "${PREDEF_GRID_NAME}",
+  obs_type = "bufr"
+ /
+
+EOF
+
+  #
+  #-----------------------------------------------------------------------
+  #
+  # link/copy executable file to working directory 
+  #
+  #-----------------------------------------------------------------------
+  #
+  exec_fn="process_Lightning.exe"
+  exec_fp="$EXECdir/${exec_fn}"
+  
+  if [ ! -f "${exec_fp}" ]; then
+    print_err_msg_exit "\
+  The executable specified in exec_fp does not exist:
+    exec_fp = \"${exec_fp}\"
+  Build lightning process and rerun."
+  fi
+  #
+  #
+  #-----------------------------------------------------------------------
+  #
+  # Run the process for lightning bufr file 
+  #
+  #-----------------------------------------------------------------------
+  #
+   PREP_STEP
+   eval $RUN_CMD_UTILS ${exec_fp} ${REDIRECT_OUT_ERR} || print_err_msg "\
+        Call to executable to run lightning process returned with nonzero exit code."
+   POST_STEP
```

Comment:
This indentation is inconsistent with the surrounding code
```suggestion
  PREP_STEP
  eval $RUN_CMD_UTILS ${exec_fp} ${REDIRECT_OUT_ERR} || print_err_msg "\
       Call to executable to run lightning process returned with nonzero exit code."
  POST_STEP
```